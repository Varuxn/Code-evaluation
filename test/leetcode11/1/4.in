93 119
9 13 177
0 56 569
36 37 222
1 5 957
61 65 879
41 58 474
14 48 401
2 11 534
30 89 516
10 27 325
39 88 563
8 64 798
1 6 312
48 80 95
39 58 915
70 45 739
3 35 619
13 16 767
24 32 25
17 44 315
3 22 417
54 59 815
53 66 162
33 79 834
82 7 687
50 90 540
32 69 496
20 24 376
14 36 332
18 41 590
70 74 198
0 1 751
71 6 629
31 47 154
18 40 787
2 39 147
16 27 73
26 34 530
29 81 392
15 2 897
17 59 270
4 15 754
12 55 138
14 49 752
22 23 147
12 91 714
2 14 761
17 21 435
38 61 677
80 84 351
73 78 982
13 30 570
1 3 472
8 38 384
2 4 774
59 37 904
39 67 304
2 9 260
21 51 634
8 28 502
36 68 400
16 52 26
1 2 802
7 62 590
15 62 645
8 10 597
52 7 543
65 77 749
22 26 447
11 57 493
18 19 31
55 22 87
32 60 840
13 17 569
74 8 11
44 45 349
58 76 367
15 71 738
35 71 609
40 82 129
46 54 520
65 73 449
51 70 301
8 25 436
21 68 18
26 29 741
32 41 277
7 43 515
18 33 589
15 92 717
12 50 546
10 18 532
66 72 824
10 87 775
91 88 965
8 12 578
26 77 687
50 53 400
46 90 305
38 51 975
12 46 221
62 73 716
63 83 16
18 20 576
31 63 207
1 31 425
84 39 377
6 8 205
10 19 674
1 7 414
86 27 177
41 86 716
33 23 256
13 42 560
61 73 930
81 85 315
60 3 982
89 16 438
12 75 702
