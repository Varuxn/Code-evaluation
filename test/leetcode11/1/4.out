0 1 2 3 5 6 7 8 9 10 11 12 13 16 18 19 20 22 23 24 25 26 27 28 30 31 33 35 36 37 38 40 41 42 43 44 45 47 48 49 50 51 52 53 56 57 59 60 61 63 64 66 68 69 70 71 72 74 75 76 78 79 80 81 82 83 84 85 86 87 89 91 92 93 96 97 98 100 101 102 103 104 105 106 107 109 110 112 113 115 117 118 

